docsets/TensorFlow_2/docset.json

```diff
@@ -9,10 +9,13 @@
     },
     "aliases": [],
     "specific_versions": [
+        {
+            "version": "2.4.0",
+            "archive": "versions/2.4.0/TensorFlow_2.tgz"
```

Comment:
Please note that you need to add the archive at this path as well. I'll add it this time.